src/test/java/com/google/firebase/auth/FirebaseUserManagerTest.java

```diff
@@ -1606,6 +1606,32 @@ public void testGetOidcProviderConfig() throws Exception {
     checkUrl(interceptor, "GET", PROJECT_BASE_URL + "/oauthIdpConfigs/oidc.provider-id");
   }
 
+  @Test
+  public void testGetOidcProviderConfigMissingId() throws Exception {
+    TestResponseInterceptor interceptor = initializeAppForUserManagement(
+        TestUtils.loadResource("oidc.json"));
+
+    try {
+      FirebaseAuth.getInstance().getOidcProviderConfig(null);
+      fail("No error thrown for missing provider ID.");
+    } catch (IllegalArgumentException e) {
+      // Expected.
+    }
+  }
+
+  @Test
+  public void testGetOidcProviderConfigInvalidId() throws Exception {
+    TestResponseInterceptor interceptor = initializeAppForUserManagement(
+        TestUtils.loadResource("oidc.json"));
+
+    try {
+      FirebaseAuth.getInstance().getOidcProviderConfig("not a valid provider ID");
```

Comment:
Nit: You can also pass a value like `saml.InvallidProviderId` here so the invalid prefix will get tested more explicitly.

Reply:
Done.